zebra-state/Cargo.toml

```diff
@@ -87,6 +87,7 @@ zebra-test = { path = "../zebra-test/", version = "1.0.1", optional = true }
 proptest = { workspace = true, optional = true }
 proptest-derive = { workspace = true, optional = true }
 derive-getters.workspace = true
+derive-new.workspace = true
```

Comment:
Nice! I like what this do with enums, generating a `new_variant()` method for each variant.